integration/test_info_cluster.py

```
@@ -3,35 +3,42 @@
 from valkey.cluster import ValkeyCluster
 from valkey.client import Valkey
 from valkeytestframework.conftest import resource_port_tracker
+from valkeytestframework.util import waiters
+from test_info_primary import _parse_info_kv_list
 
-def _parse_info_kv_list(reply):
-    it = iter(reply)
-    out = {}
-    for k in it:
-        v = next(it, None)
-        out[k.decode() if isinstance(k, bytes) else k] = v
-    return out
+class TestFTInfoCluster(ValkeySearchClusterTestCase):
 
-class TestFTInfoClusterCompleted(ValkeySearchClusterTestCase):
+    def is_indexing_complete(self, node, index_name):
+        try:
+            raw = node.execute_command("FT.INFO", index_name, "CLUSTER")
+            info = _parse_info_kv_list(raw)
+            if not info:
+                return False
+            backfill_in_progress = int(info.get("backfill_in_progress", 1))
+            state = info.get("state", "")
+            return backfill_in_progress == 0 and state == "ready"
+        except:
```

This kind of general exception isn't good practice as it can hide failures that aren't expected. If you really are expecting an exception, then you should the type and compare the detailed string to ensure that you're catching the exception your expecting. If you aren't expecting an exception, then don't use the try/except, let it crash -- you have bug somewhere.